scala-mode-syntax.el

```diff
@@ -600,6 +600,16 @@ symbol constituents (syntax 3)."
                  (scala-syntax:put-syntax-table-property 0 '(3 . nil)))
                '(3 . nil))))))))) ;; symbol constituent syntax (3) also for the '_'
 
+(defun scala-syntax:propertize-special-symbols (start end)
+  (save-excursion
+     (goto-char start)
+     (while (re-search-forward (concat "[" scala-syntax:opchar-group "]" scala-syntax:op-re) end t)
+       (let ((match-beg (match-beginning 0))
+             (match-end (match-end 0))
+             (match (match-string 0)))
+         (unless (member match '("/*" "//" "/**" "</" "*/"))
```

Comment:
this is not a problem only of reserved symbols.

```scala
List(1)|+|List(2)
```
will mark `|+|List` as a single symbol as well.

What we can do is to set consecutive operator characters as symbols only if they are delimiter by space (or by punctuation in general)